staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -946,6 +946,19 @@ type VirtualMachineStatus struct {
 	// StateChangeRequests indicates a list of actions that should be taken on a VMI
 	// e.g. stop a specific VMI then start a new one.
 	StateChangeRequests []VirtualMachineStateChangeRequest `json:"stateChangeRequests,omitempty" optional:"true"`
+	// VolumeSnapshotStatuses indicates a list of statuses whether snapshotting is
+	// supported by each volume.
+	VolumeSnapshotStatuses []VolumeSnapshotStatus `json:"volumeSnapshotStatuses,omitempty" optional:"true"`
```

Comment:
should we add the +listType here instead? https://kubernetes.io/docs/reference/using-api/api-concepts/#merge-strategy

Reply:
Yeah, I probably should. I'm not sure which would be appropriate, `// +listType=map` or rather `// +listType=atomic`. What do you think?